src/dmd/dsymbolsem.d

```diff
@@ -1148,8 +1148,14 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             {
                 AliasDeclaration ad = imp.aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
-                if (imp.mod.search(imp.loc, imp.names[i]))
+                Dsymbol importedSymbol = imp.mod.search(imp.loc, imp.names[i]);
+                if (importedSymbol)
                 {
+                    // @@@DEPRECATED_2018-01@@@
```

Comment:
As I tried to explain to @JinShil, this needs to be `2019` instead of `2018` as it should be the date when the maintenance act is needed.
See e.g. https://github.com/wilzbach/phobos/blob/complex/std/datetime/timezone.d#L145

Reply:
Also, please add some explanatory text such.  Something like `Deprecated in 2018-01.  Change to error in 2019-01.  @@@DEPRECATED_2019-01@@@`.

Reply:
Done, thanks.